docs/deployment.md

```diff
@@ -23,7 +32,33 @@ resource_registry:
   OS::TripleO::Services::NovaCompute: OS::Heat::None
 
 parameter_defaults:
-  NeutronMechanismDrivers: [openvswitch, baremetal]
+  NeutronMechanismDrivers: [ovn, baremetal]
+  IronicEnabledHardwareTypes:
+  - ipmi
+  IronicEnabledPowerInterfaces:
+  - ipmitool
+  IronicEnabledManagementInterfaces:
+  - ipmitool
+  IronicCleaningDiskErase: 'metadata'
+  IronicInspectorSubnets:
+  - ip_range: 192.168.1.200,192.168.1.250
+  IronicInspectorInterface: 'br-ctlplane'
+```
+
+A sample `standalone_parameters.yaml` file with all other parameters looks like this.
+Note that `NeutronMechanismDrivers` is set to `ovn` and `baremetal`.
```

Comment:
It's probably worth mentioning what this parameter does and/or why these particular values are important.